processor/src/main/java/org/mapstruct/ap/internal/conversion/BigDecimalToStringConversion.java

```diff
@@ -19,15 +19,16 @@
 package org.mapstruct.ap.internal.conversion;
 
```

Comment:
@gunnarmorling I think that is because there is no `PACKAGE_DEF` (no check after package declaration) in the tokens of the `EmptyLineSeparator` check